styles/style.css

```diff
@@ -0,0 +1,699 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'montserrat';
+  }
+
+a {
+	color: #000000;
+  text-decoration: none;
+}
+
+.wrapper  {
+  width: 100%;
+  height: 100%;
+  min-width: 370px;
+  max-width: 1440px;
+  margin: 0 auto;
+}
+
+.header {
+  margin-bottom: 31px;
+  white-space: nowrap;;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 26px 40px 0 46px;
+  margin-bottom: 42px;
+}
+
+.logo {
+  margin-right: 2vw;
+}
+
+.header-models {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-size: 15px;
+  font-weight: 600;
+  font-style: normal;
+  white-space: nowrap;
+}
+
+.model {
+  margin: 0 32px 6px 0;
+}
+
+.model:hover {
+  color: #E51937;
+  border-bottom: 2px solid;
+  padding: 0 0 6px 0;
+}
+
+.model-s__link {
+  color: #E51937;
+  border-bottom: 2px solid;
+  padding: 0 0 6px 0;
+}
+
+.header-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 15px;
+  font-weight: 600;
+  font-style: normal;
+  white-space: nowrap;
+}
+
+.header-menu__item {
+  margin: 0 32px 6px 0;
+}
+
+.header-menu__item:hover {
+  color: #E51937;
+  border-bottom: 2px solid;
+  padding: 0 0 6px 0;
+}
+
+.header-content {
+  display: flex;
+  flex-flow: column;
+}
+
+.first-floor__item {
+  display: flex;
+  flex-flow: column;
+  text-align: center;
+  background-image: url(../img/Tesla-Wordmark-Red-1.png);
+  background-repeat: no-repeat;
+  background-size: 101%;
+  background-position-y: 5%;
+}
+
+.first-floor__title {
+  display: flex;
+  flex-flow: column;
+  margin: 0 auto;
+}
+
+.title-text {
+  font-size: 89px;
+  font-weight: 600;
+  margin: 0 auto;
+  padding-bottom: 16px;
+  position: relative;
+  top: 11px;
+  left: 9px;
+  letter-spacing: .6px;
+}
+
+.title-subtext {
+  font-size: 15px;
+  font-weight: 500;
+  font-style: normal;
+  margin: 0 auto;
+  position: relative;
+  bottom: 12px;
+  left: 9px;
+}
+
+.car-tesla {
+  width: 68%;
+  margin: 0 auto;
+  margin-bottom: 38px;
+  position: relative;
+  top: 15px;
+  right: 24px;
+}
+
+.stat {
+  width: 700px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  margin-bottom: 53px;
+}
+
+.params {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 100%;
+  padding: 0 300px 0 382px;
+}
+
+.wrap-params-one {
+  display: flex;
+  align-items: center;
+}
+
+.stat__speed {
+  width: 46px;
+  height: 35px;
+  margin-right: 187px;
+}
+
+.stat-power {
+  width: 46px;
+  height: 35px;
+  margin-right: 214px;
+}
+
+.wrap-params-two {
+  display: flex;
+  align-items: center;
+}
+
+.stat__test-drive {
+  width: 87px;
+  height: 0;
+  margin-right: 115px;
+  position: relative;
+  right: 12px;
+}
+
+.test-drive__red {
+  display: none;
+}
+
+.stat__btn {
+  position: relative;
+  top: 11px;
+  right: 53px;
+}
+ 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)